root/programs/MultiPurposeAuthSite/MultiPurposeAuthSite/Models/ASPNETIdentity/ASPNETIdentityConfig.cs

```diff
@@ -0,0 +1,1324 @@
+﻿//**********************************************************************************
+//* Copyright (C) 2007,2016 Hitachi Solutions,Ltd.
+//**********************************************************************************
+
+#region Apache License
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. 
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#endregion
+
+//**********************************************************************************
+//* クラス名        ：ASPNETIdentityConfig
+//* クラス日本語名  ：ASP.NET IdentityのConfigクラス（ライブラリ）
+//*
+//* 作成日時        ：－
+//* 作成者          ：－
+//* 更新履歴        ：－
+//*
+//*  日時        更新者            内容
+//*  ----------  ----------------  -------------------------------------------------
+//*  2017/04/24  西野 大介         新規
+//**********************************************************************************
+
+using System;
+using System.Configuration;
+
+/// <summary>MultiPurposeAuthSite.Models.ASPNETIdentity</summary>
+namespace MultiPurposeAuthSite.Models.ASPNETIdentity
+{
+    /// <summary>列挙型</summary>
+    public enum EnumUserStoreType
+    {
+        /// <summary>Memory Provider</summary>
+        Memory,
+        /// <summary>DBMS Provider (SqlServer)</summary>
+        SqlServer,
+        /// <summary>DBMS Provider (Oracle Manage Driver)</summary>
+        OracleMD,
```

Comment:
@daisukenishino 

The name of members should be easy to understand.

For example:  
ODPNETManaged  
ODPManaged  
ODPManagedDriver